src/python/pants/backend/BUILD

```diff
@@ -2,3 +2,116 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 python_sources()
+
+
+# Setup rules about which parts of the codebase may depend on backend sources.
+
+DEFAULT_BACKEND_RULES = (
```

Comment:
These aren't full rules, right, just the list of allowed dependents? this should maybe be named DEFAULT_ALLOWED_DEPENDENTS or something? (The BACKEND part seems unnecessary, since we're in the backend subdir?)